client/src/styles/RecipePage.css

```diff
@@ -0,0 +1,148 @@
+.recipe-page-container {
+  margin: 30px;
+}
+
+.icon-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;;
+}
+.icon-button {
+  height: 40px;
+  width: 40px;
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+  background: none;
+  border: 2px solid black;
+  border-radius: 50%;
+  -moz-border-radius: 50%;
+  -webkit-border-radius: 50%;
+
+}
+
+.icon {
+  margin: 0 auto;
+  display: block;
+}
+
+.recipe-title {
+  word-wrap: break-word;
+}
+
+.info-container {
+  border-top: 2px solid black;
+  border-bottom: 2px solid black;
+  padding: 30px 20px 10px 20px;
+  padding-right: 35px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 30px;
+  gap: 20px;
+}
+
+.icon-background.active.no-pointer {
+  cursor:auto;
+}
+
+.rating-container {
+    width: 75px;
+    min-width: 75px;
+    height: 110px;
+    padding-top: 5px;
+}
+
+.star-rating {
+  width: 25px;
+  height: 25px;
+}
+
+.star-rating.filled {
+    fill: var(--yellow);
+}
+
+.attributes-container {
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+}
+
+.attribute-text {
+    font-size: 16px;
+    margin: 0px;
+    margin-bottom: 5px;
+    color: var(--black);
+    font-family: 'Open Sans';
+    word-wrap: break-word;
+  }
+
+.primary-btn.cook-now {
+    margin-top: 30px;
+    margin-bottom: 100px;
+}
+
+.category-container {
+    display: flex;
+    flex-direction: row;
+    overflow: scroll;
```

Comment:
Add
```
  overflow-y: hidden;
  height: 130px;
```
to fix scrolling

Reply:
Overflow-y hidden but left height at 110px